strawberry/middleware.py

```diff
@@ -1,13 +1,19 @@
 from .directive import DIRECTIVE_REGISTRY
 
 
+SPECIFIED_DIRECTIVES = {"include", "skip"}
+
+
 class DirectivesMiddleware:
     def resolve(self, next_, root, info, **kwargs):
         result = next_(root, info, **kwargs)
 
         for directive in info.field_nodes[0].directives:
             directive_name = directive.name.value
 
+            if directive_name in SPECIFIED_DIRECTIVES:
+                continue
```

Comment:
can we add a comment on why we do this? 

Do you have any other possible names for SPECIFIED_DIRECTIVES? Not sure what it means right now ;)

Reply:
`DirectivesMiddleware` tries to find resolve function for `skip` and `include` directives,  but `DIRECTIVE_REGISTRY` is empty when app startup. `graphql-core-next` has own logic for these specified directives [here](https://github.com/graphql-python/graphql-core-next/blob/master/src/graphql/execution/execute.py#L513), so we can skip them.

I used this name because `graphql-python` [uses](https://github.com/graphql-python/graphql-core-next/blob/master/src/graphql/type/directives.py#L200) it. Maybe we can use better name, I'm not sure about this. Also, probably we can dynamically create this set, like:

```
from graphql.type.directives import specified_directives

SPECIFIED_DIRECTIVES = {directive.name for directive in specified_directives}
```

Reply:
ah good, point 😊 let's keep it like that for now!